agents/skala-r/fence_skalar.py

```diff
@@ -206,6 +201,7 @@ def main():
     
     if "--ssl" in options or "--ssl-secure" in options or "--ssl-insecure" in options:
         proto = "https://"
+        ssl_verify = True
     else:
         proto = "http://"
         ssl_verify = False
```

Comment:
You probably want this in an `elif "--ssl-insecure" in options:` as it only affects SSL/TLS requests, and you could bump `https://` outside the if/else as I dont think anyone wants to use things without SSL these days.

Reply:
Fixed.

>I dont think anyone wants to use things without SSL these days.
You would be suprised, but yes sometimes someone wants to go without SSL.